src/components/UI/Dropdown/index.js

```diff
@@ -1,19 +1,101 @@
-import React from "react";
-import i18next from "i18next";
-// Styles
-import { Container } from "./dropdown.styled";
-
-export function Dropdown() {
-  const handleLanguageChange = (lang = "en") => {
-    i18next.changeLanguage("sr");
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { IoIosArrowDown } from 'react-icons/io';
+
+export function Dropdown(props) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [handleOption, setHandleOption] = useState(props?.list[0] || []);
```

Comment:
`selectedOption` & `setSelectedOption` is much clearer
you should never name a variable starting with a verb, because when we see a verb we expect a method